test-project/build.gradle.kts

```diff
@@ -1,15 +0,0 @@
-
-abstract class MyTask : DefaultTask() {
```

Comment:
This file is important since it is used to show how the IDE support works.

Reply:
We can move it to the other build logic files, though it seems to be more distracting there. And if you don't want to look into these changes here, then you can ignore this file completely. See the video I recorded what we show with these changes.

Reply:
My only problem is that it's in the root of `test-project` and when I was first looking at the repo, it got me pretty confused as to why it's here. If we could move it to some other directory, that'd already be an improvement.

Reply:
If we could make this file part of the build somehow, then the video could be made better, too, I think. Now there is a big break between the first part and the second part for me.

Reply:
Thinking about this again, we can also remove it. This was about showing the IDE support for the assign operator better, though we'll get better IDE support and then we can show it directly in the already existing Kotlin build scripts.